next.config.js

```diff
@@ -352,8 +352,16 @@ async function redirects() {
     },
     { source: '/org/terms.html', destination: '/references/glossary', permanent: true },
     // overview redirects
-    { source: '/stacks-blockchain', destination: '/stacks-blockchain/overview', permanent: true },
```

Comment:
we shouldn't change existing redirect sources. For these particular pages, we should set the destination to the new URL